projects/packages/jetpack-mu-wpcom/src/features/wpcom-dashboard-widgets/wpcom-launchpad-widget/index.js

```diff
@@ -0,0 +1,53 @@
+import { Launchpad } from '@automattic/launchpad';
+import { QueryClient, QueryClientProvider } from '@tanstack/react-query';
+import { useRefEffect } from '@wordpress/compose';
+
+import './style.scss';
+
+const queryClient = new QueryClient();
+
+/**
+ * Set the href base of all relative links to the wordpress.com.
+ *
+ * @return {Function} A ref callback.
+ */
+function useSetHrefBase() {
```

Comment:
What is this weird hook

Reply:
So currently _some_ of the links in `Launchpad` are relative. These are supposed to be wordpress.com links, so we need to adjust the base. There other wp-admin links are absolute and they are correct.

Reply:
Can we instead have a prop for the component instead (maybe as a follow-up) to make the component calypso agnostic.